doc/generate-kuadrant-auth-policy.md

```diff
@@ -38,24 +42,20 @@ components:
   securitySchemes:
     securedDog:
       type: openIdConnect
-      openIdConnectUrl: https://example.com/.well-known/openid-configuration
+      openIdConnectUrl: https://sso.example.com/auth/realms/petstore
 ```
 
-Running the command
+Take this example and save it as `example.yaml` and than run the command:
```

Comment:
```suggestion
Take this example and save it as `example.yaml` and then run the command:
```

Reply:
English is not my first language, thanks!